### PR TITLE
Fixing the auto update

### PR DESCRIPTION
**Pull request by avsthiago**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
> I don't think that this stopping the bot from updating the version.

I think so too. I just wanted to make sure I could rebuild the 4.23.1. Now I'm working on what might be blocking the bot. code-server doesn't build for Mac Intel anymore. The last commit is my first try on fixing that.

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
@conda-forge-admin, please rerender

**Comment by avsthiago** [comment]
Hi @xhochy, could you have a look at this fix?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
+  url: https://github.com/coder/code-server/releases/download/v{{ version }}/code-server-{{ version }}-macos-arm64.tar.gz  # [osx]
+  sha256: 6f5e4ca39d0ae7a609707fd9c761247be1ed9e33147372320a84d30c3df336c3  # [osx]
   folder: code-server
 
 build:
-  number: 1
+  number: 0
   binary_relocation: false
-  skip: true  # [win]
+  skip: true  # [win or target_platform == 'osx-64']
```
Comment (xhochy) [inline recipe/meta.yaml:20]:
```suggestion
  skip: true  # [win or (osx and x86_64)]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
-  url: https://github.com/cdr/code-server/releases/download/v{{ version }}/code-server-{{ version }}-linux-arm64.tar.gz  # [aarch64]
-  sha256: d2e18d03a2434bca7f70a25870df3d571283582172410c3ce05b1bb5bba1a71b  # [aarch64]
-  url: https://github.com/cdr/code-server/releases/download/v{{ version }}/code-server-{{ version }}-macos-amd64.tar.gz  # [osx]
-  sha256: 152e3bc287cec6187a832c3970e17dd201671501ca26b6b48355a0e10e1276b6  # [osx]
+  url: https://github.com/coder/code-server/releases/download/v{{ version }}/code-server-{{ version }}-linux-amd64.tar.gz  # [linux64]
+  sha256: 7b60ae570dc8a4a5e56ddfdd73a7cce40b2b155b390dffa6897f815e9510a0eb  # [linux64]
+  url: https://github.com/coder/code-server/releases/download/v{{ version }}/code-server-{{ version }}-linux-arm64.tar.gz  # [aarch64]
+  sha256: a07b7640854619e36ecca75fe0b240c1f671e00f9c08eb2b1cb7f06bcbf39dde  # [aarch64]
+  url: https://github.com/coder/code-server/releases/download/v{{ version }}/code-server-{{ version }}-macos-arm64.tar.gz  # [osx]
+  sha256: 6f5e4ca39d0ae7a609707fd9c761247be1ed9e33147372320a84d30c3df336c3  # [osx]
```
Comment (xhochy) [inline recipe/meta.yaml:14]:
```suggestion
  url: https://github.com/coder/code-server/releases/download/v{{ version }}/code-server-{{ version }}-macos-arm64.tar.gz  # [osx and arm64]
  sha256: 6f5e4ca39d0ae7a609707fd9c761247be1ed9e33147372320a84d30c3df336c3  # [osx and arm64]
```

**Review by xhochy — APPROVED** [review]

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

**Pull request merged**